lisa/sut_orchestrator/azure/features.py

```diff
@@ -1475,6 +1509,34 @@ def _call_requirement_method(
             schema.disk_controller_type_priority,
         )
 
+        # refer
+        # https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmssstorageprofile?view=azps-13.0.0               # noqa: E501
```

Comment:
It's good to have the urls, please summarize to avoid people must visit links to know details. In case the links are expired, nothing can help understand it.